packages/components/src/components/buttons/Button/index.tsx

```diff
@@ -86,7 +86,6 @@ const Wrapper = styled.button<Props>`
     ${props =>
         props.isInverse &&
         !props.isDisabled &&
-        !props.isLoading &&
```

Comment:
this will be "breaking change" for the wallet, I remember there was some reason we did it this way (something about when button is in loading state it should be "activated"?), but maybe this new behaviour make more sense. After all we can manually switch the isInverse prop on app side if we need to so let's try it 👍

Reply:
This is actually a bug fix i think. Because when you had inverse button and you set it to loading state the style changed - which was weird.
If you want different style on loading you can still do it in your app simply by changing the variant when loading.